x/auth/ante/basic.go

```diff
@@ -79,9 +80,29 @@ type ConsumeTxSizeGasDecorator struct {
 	ak AccountKeeper
 }
 
-func NewConsumeGasForTxSizeDecorator(ak AccountKeeper) ConsumeTxSizeGasDecorator {
-	return ConsumeTxSizeGasDecorator{
-		ak: ak,
+type ConsumeTxSizeGasDepDecorator struct {}
+
+func (d ConsumeTxSizeGasDepDecorator) AnteDeps(txDeps []sdkacltypes.AccessOperation, tx sdk.Tx, next sdk.AnteDepGenerator) (newTxDeps []sdkacltypes.AccessOperation, err error) {
```

Comment:
We can just implement this for `ConsumeTxSizeGasDecorator` and then we don't have to specifically wrap them together.